test/run-tests.c

```diff
@@ -75,6 +75,16 @@ int main(int argc, char **argv) {
   platform_init(argc, argv);
   argv = uv_setup_args(argc, argv);
 
+  if ((argc == 2 || argc == 4) && strcmp(argv[1], "spawn_helper5") == 0) {
+    uv__check_nfd(4);
+  } else if ((argc == 2 || argc == 4) && strcmp(argv[1], "spawn_tcp_server_helper") == 0) {
+    uv__check_nfd(4);
+  } else if ((argc == 2 || argc == 4) && strcmp(argv[1], "spawn_helper8") == 0) {
+    /* skip */
+  } else {
+    uv__check_nfd(3);
+  }
```

Comment:
I haven't seen any issues with gdb or lldb locally, but I will note that the GitHub Actions CI system test harness does appear have a number of problems with excess file descriptors, so it might be bad to merge this for that reason.